resources/views/auth/info.blade.php

```diff
@@ -0,0 +1,159 @@
+                                    <div class="col-sm-8">
+                                    <input class="form-control" type="text" name="year_of_the_leaving_exam"
+                                           id="year_of_the_leaving_exam"
+                                           required>
+                                    </div>
+                                </div>
+                                <div class="form-group">
+                                    <label class="col-sm-4 control" for="neptun_identifier">@lang('user.neptun_identifier')</label>
+                                    <div class="col-sm-8">
+                                    <input class="form-control" type="text" name="neptun_identifier"
+                                           id="neptun_identifier" required>
+                                    </div>
+                                </div>
+                                <div class="form-group">
+                                    <label class="col-sm-4 control" for="year_of_enrollment">@lang('user.year_of_enrollment')</label>
+                                    <div class="col-sm-8">
+                                    <input class="form-control" type="text" name="year_of_enrollment"
+                                           id="year_of_enrollment" required>
+                                    </div>
+                                </div>
+                                <label class="col-sm-4 control" for="faculty">@lang('user.faculty')</label>
+                                <div class="col-sm-8">
+                                <select id="faculty" class="form-control">
+                                    <option value="jogtudomanyi_kar">@lang('user.jogtudomanyi_kar')</option>
+                                    <option value="barczi_gusztav_gyogypedagogiai_kar">@lang('user.barczi_gusztav_gyogypedagogiai_kar')</option>
+                                    <option value="bolcseszettudomanyi_kar">@lang('user.bolcseszettudomanyi_kar')</option>
+                                    <option value="pedagogiai_es_pszichologiai_kar">@lang('user.pedagogiai_es_pszichologiai_kar')</option>
+                                    <option value="tanito_eso_vokepzo_kar">@lang('user.tanito_eso_vokepzo_kar')</option>
+                                    <option value="tarsadalomtudomanyi_kar">@lang('user.tarsadalomtudomanyi_kar')</option>
+                                    <option value="termeszettudomanyi_kar">@lang('user.termeszettudomanyi_kar')</option>
+                                </select>
+                                </div>
+                                <div>
+                                    <label class="col-sm-4 control" for="workshop">@lang('user.workshop')</label>
+                                    <div class="col-sm-8">
+                                    <select multiple class="form-control" id="workshop">
```

Comment:
Since a student can be part of multiple workshops in some cases, it would be sensible to have a multi-select here.

For examples see [this](https://developer.snapappointments.com/bootstrap-select/examples).
But most likely something like this:
```html
<select class="selectpicker" multiple>
  <option>Mustard</option>
  <option>Ketchup</option>
  <option>Relish</option>
</select>
```

Reply:
this not working

Reply:
I still think we should use such `selectpicker` since makes the UI more self-explanatory.
Maybe some Bootstrap hackery is required to accomplish this.

I'm looking forward to the solution, maybe @katkodominik can help with this, since he used some different Bootstrap selector.

Reply:
I try resoled this problem but  didn´t work this oportunity. I used javascript from library and other files also didnt work. I can't solve this problem.